src/client/theme-default/components/VPNavScreenTranslations.vue

```diff
@@ -1,10 +1,12 @@
 <script setup lang="ts">
-import { ref } from 'vue'
+import { computed, ref } from 'vue'
 import { useLangs } from '../composables/langs'
 import VPLink from './VPLink.vue'
+import VPSocialLink from "./VPSocialLink.vue";
 
 const { localeLinks, currentLang } = useLangs({ correspondingLink: true })
 const isOpen = ref(false)
+const repo = computed(() => !!currentLang.value.repository || (localeLinks.value.length > 1 && localeLinks.value.some(l => !!l.repository)))
```

Comment:
this should be `localeLinks.value.length && localeLinks.value.some(l => !!l.repository)`